index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+      <!-- <h1>hello word!, we are software developers!</h1> -->
+      <h1 id="title">Survey Form</h1>
+      <p id="description">This is a survey form to submmit to the bootcamp</p>
+      <form id="survey-form" >
+        <div>
+          <label id="name-label">Name</label>
+
+          <input id='name' type='text' required placeholder="Enter your name" />
+        </div>
+        
+        <div>
+          <label id="email-label">Email</label>
+          <input type="email" name="Email" id="email" required placeholder="Enter your Email" />
+        </div>
+
+        <div>
+          <label id="number-label">Age</label>
+          <input id="number" type="number" min="16" max="100" placeholder="Age"/>
+        </div>
+
+        <select id="dropdown">
+          <option disabled selected value>Select current role</option>
+          <option>Student</option>
+          <option>Full time job</option>
+        </select>
+
+        <div>
+          <p>Would you recommend freeCodeCamp to a friend?</p>
+          <input type="radio" name="recommend" value="Definitely">
+          <label>Definitely</label>
+
+          
+          <input type="radio" name="recommend" value="Maybe">
+          <label>Maybe</label>
+        </div>
+
+        <div>
+          <p>What would you like to see improved? (Check all that apply)</p>
+          <input type="checkbox" value="Front-End">
+          <label>Front-End Projects</label>
+
+          
+          <input type="checkbox" value="Back-End">
+          <label>Back-End Projects</label>
+        </div>
+
+        <div>
+          <p>Any comments or suggestions?</p>
+          <textarea placeholder="Enter your comment here" ></textarea>
+        </div>
+
+        <button  id="submit">Submit</button>
+      </form>
```

Comment:
The form could be contained in a section tag

Reply:
- [ ] It is important that for every comment you add you should as well go deep by explaining the `why` a change is being requested. This would assist the students to better understand the reason for a change and they know how best to go about it in their next projects :+1: :sparkles: 
- [ ] please i kindly suggest you try to sound polite words when requesting for changes from the student, you can use words such as `i kindly suggest, i think you can, please ensure you` :+1: :sparkles: